README.md

```diff
@@ -136,6 +137,33 @@ create extension pg_net;
 
 ---
 
+# Extension Configuration:
+
+the extension creates 3 configurable variables:
+
+1. **pg_net.batch_size** _(default: 200)_: An integer that limits the max number of rows that the extension will process from _`net.http_request_queue`_ during each read
+2. **pg_net.ttl** _(default: 6 hours)_: An interval that defines the max time a row in the _`net.http_response`_ will live before being deleted
+3. **pg_net.database_name** _(default: 'postgres')_: A string that defines which database the extension is applied to
+
+All these variables can be viewed with the following command:
+```sql
+select  * from pg_settings WHERE name LIKE 'pg_net%'
+```
+
+The postgres.conf file can be found with the following SQL command:
+```sql
+SHOW config_file;
+```
+
+You can change the variables by adding any of the following line to your postgres.conf file
+```
+pg_net.batch_size = <new integer>
+pg_net.ttl = '<new ttl interval>'
+pg_net.database_name = '<database name>'
+```
+
+After saving the file, you can execute `SELECT pg_reload_conf()` to update _postgres.conf_ for your database. If the extension does not respond to the update, it may be necessary to restart your database.
```

Comment:
@TheOtherBrian1 By any chance did you tried doing this on Supabase? Or do you self host?

I think on Supabase right now you can't do `ALTER SYSTEM ..` plus `pg_reload_conf()` since they require superuser.

Reply:
I tested this on a self hosted instance. I just ran the following SQL to see how Supabase would react:

```sql
ALTER SYSTEM SET pg_net.batch_size = 200;
```

It gave an error:

> Error running SQL: Failed to run sql query: permission denied to set parameter "pg_net.batch_size"

Through Supabase's CLI, it is possible to override some config variables, but the [permitted list](https://supabase.com/docs/guides/platform/custom-postgres-config) does not include PG_NET variables.

I'll add a caveat to the documentation that configuration is only possible on self-hosted instances.